Cirrious.FluentLayout/AdvancedFluentLayoutExtensions.cs

```diff
@@ -4,9 +4,12 @@
 // Contributions and inspirations noted in readme.md and license.txt
 // 
 // Project Lead - Stuart Lodge, @slodge, me@slodge.com
+//
+//Softlion: VerticalStackPanelConstraints auto height panel view (ie: parent view) in all cases !
```

Comment:
lol - but I am happy to move this repo somewhere else (outside my personal account) and put up wider attribution....

If in doubt, assume reshaper added and maintained file copyright headers :)